### PR TITLE
[Tests] Rest Load test

### PR DESCRIPTION
**Pull request by dkijania**
Simple load test for 2 rest endpoints (health,proposals).

command:
`cargo test load --features non-functional`

### CODE REVIEW

File: vit-servicing-station-tests/src/common/clients/rest.rs [hunk 1]
```diff
@@ -25,37 +65,59 @@ impl RestClient {
...
+        Ok(response)
+    }
+
+    pub fn disable_log(&mut self) {
+        self.logger.set_enabled(false);
+    }
+
     fn path(&self, path: &str) -> String {
-        format!("http://{}/api/v0/{}", self.address, path)
+        let path = format!("http://{}/api/v0/{}", self.address, path);
```
Comment (danielSanchezQ) [inline vit-servicing-station-tests/src/common/clients/rest.rs:118]: Didn't we have a path builder?

File: vit-servicing-station-tests/src/tests/mod.rs [hunk 2]
```diff
@@ -1,3 +1,5 @@
 pub mod bootstrap;
 pub mod graphql;
+#[cfg(feature = "non-functional")]
+pub mod non_functional;
```
Comment (danielSanchezQ) [inline vit-servicing-station-tests/src/tests/mod.rs:4]: Why do we want to activate/deactivate this testing? Is this gonna run in some other runners?

File: vit-servicing-station-tests/src/common/clients/rest.rs (hunk 1 above)
Comment (dkijania) [reply vit-servicing-station-tests/src/common/clients/rest.rs:118]: In another pr which is not merged yet

File: vit-servicing-station-tests/src/tests/mod.rs (hunk 2 above)
Comment (dkijania) [reply vit-servicing-station-tests/src/tests/mod.rs:4]: usually non_functional tests are a bit longer (up to 24 hours). so we probably don't want to run them during CI

Comment (danielSanchezQ) [reply vit-servicing-station-tests/src/tests/mod.rs:4]: Got it. Thanks!

File: vit-servicing-station-tests/src/common/clients/rest.rs (hunk 1 above)
Comment (danielSanchezQ) [reply vit-servicing-station-tests/src/common/clients/rest.rs:118]: Cool, we will migrate this piece to the builder later then.

**Pull request merged**